lib/jekyll/commands/new.rb

```diff
@@ -21,6 +22,12 @@ def init_with_program(prog)
         def process(args, options = {})
           raise ArgumentError, "You must specify a path." if args.empty?
 
+          if args.size == 1
+            new_blog_title = args[0]
```

Comment:
@ashmaroli 
![hug it out](https://media0.giphy.com/media/l2JJJHGbjqUDZJp8A/200w.gif)